### PR TITLE
Update User guide with methods accuracy & metrics time calculation

### PR DESCRIPTION
**Pull request by GalyaZalesskaya**

### CODE REVIEW

File: docs/source/user-guide.md [hunk 1]
```diff
@@ -392,6 +393,25 @@ Target layer is the part of the model graph where XAI branch will be inserted (a
 
 All supported methods are gradient-free, which suits deployment framework settings (e.g. OpenVINO™), where the model is in optimized or compiled representation.
 
+## Methods performance/accuracy comparison:
+
+The table below compares accuracy and performace of different models and explain methods (learn more about [Quality Metrics](#measure-quality-metrics-of-saliency-maps)):
+
+|            Model            | Explain mode | Explain method | Pointing game |   | Insertion | Deletion |   Delta  |   |   ADCC   | Coherency | Complexity | Average Drop | #Model inferences |
```
Comment (GalyaZalesskaya) [inline docs/source/user-guide.md]: If needed, I can reduce the table, leaving just `Pointing game`, `Insertion`, `Deletion` and `ADCC` metrics

File: docs/source/user-guide.md
Comment (goodsong81) [inline docs/source/user-guide.md:1]: We may want to consolidate `#Model inferences` and `Explain time` terms.

**Review by goodsong81 — COMMENTED** [review]
It looks OK to me but it would be better to specify the dataset info like 10% sampling from ImageNet, etc.

File: docs/source/user-guide.md
Comment (GalyaZalesskaya) [reply docs/source/user-guide.md:1]: Thank you, applied in [91b5936](https://github.com/openvinotoolkit/openvino_xai/pull/76/commits/91b5936ae72eba15644ae9df43c21c34e3ad51b6)

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**